specs/import-cluster-view.adoc

```diff
@@ -0,0 +1,133 @@
+= Import cluster workflow by using tendrl-frontend UI
+
+The intent of import cluster workflow is to provide a mechanism to add an existing storage cluster(gluster, ceph etc) into tendrl for management using tendrl-frontend UI.
+
+* Assumption: Only 1 cluster can be imported at a time using the Import Cluster workflow.
+
+== Problem description
+
+This specification describes import cluster workflow using UI which is a mechanism to add an existing storage cluster(gluster, ceph etc) into tendrl for management. This specification addresses the import use-cases of both ceph and gluster storage.
+
+== Use Cases
+
+This addresses the use case of importing existing clusters(ceph. gluster etc.) into tendrl for management.
+
+The workflow is as follows:
+
+* "Import Cluster" can be triggered from 
+
+	- Landing page (First Time Experience) if no cluster is present in tendrl
+	  (https://redhat.invisionapp.com/share/6T900V2ZX#/screens/198042644)
+
+	- Cluster List View if minimum one cluster is present in tendrl.
+	  (https://redhat.invisionapp.com/share/BR8JDCGSQ#/screens/185937524)
+
+* User clicks on "Import Cluster". 
+
+* It will take to host list page where user will select the hosts he wants to import.
```

Comment:
User selects a single host participating in a cluster. Upon clicking 'Next" the API gets a list of hosts participating in the cluster. I think we need to reword